applications/ShapeOptimizationApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -72,28 +72,28 @@ inline void InverseMapScalar(TMapper& mapper,
     mapper.InverseMap(origin_variable, destination_variable);
 }
 
-inline double ComputeL2NormScalar(OptimizationUtilities& utils, const Variable< double >& variable)
+inline double ComputeL2NormScalar(ModelPart& rModelPart, const Variable< double >& variable)
```

Comment:
I was thinking about making those also static? Would it be too much work?

Reply:
moved to lambdas now